### DISCUSSION TITLE
BUG: Bind phrase won't change UID if firmware is built locally

### DISCUSSION THREAD

**Discussion by Z-Vladus** [ExpressLRS/ExpressLRS#2776] [General]
Hello!
I have encountered the problem. 
I have pulled latest changes to my local repository and have built and flashed my 2 receivers : BetaFPV 900 RX and iFlight 2400 RX. Now, web UI does not change the UID below when the passphrase is typed, so I am unable to change UID via web interface.
I have tried clean building by downloading the v3.4.2 zip from Releases section, the problem persists.
Then I have tried v3.4.1 source code, and it worked fine, it builds, flashes, phrase changes the UID.
Confusing thing is, the v3.4.2 via Configurator generates the working firmware file and it works fine.
The TX is not affected by this bug.

**Comment by pkendall64** [comment]
I think you will find that the page has not fully loaded so the javascript is not running.

**Reply by Z-Vladus** [reply]
I doubt that.
The same notebook, the same IDE, the same browser, but different results.
Anyways, thanks for the reply and for pointing to the probable problem. 
I think I will stay with 3.4.1 for some time. Maybe someone will encounter the same issue and it will be clear how to fix.

**Comment by Ncerzzk** [comment]
I think https://github.com/ExpressLRS/ExpressLRS/pull/2756 this pr fixed your problem, especially this change:
```
data.pwm?.forEach((item,index) => {
```
